### DISCUSSION TITLE
Language files don't work

### DISCUSSION THREAD

**Discussion by hugeBlack** [FabricMC/fabric-api#1330] [Mod Dev Support] [answered]
I use TranslatableText to display texts in my language files and it works well in development environment. However, when I install my mod in mods folder, the language files don't work and it just displays the key of TranslatableText and I can't see any error about it. And I'm sure the files are in the right position.

![20210223235710](https://user-images.githubusercontent.com/60165378/108870252-e8bbdb80-7632-11eb-91c1-dee937f37001.jpg)

**Comment by haykam821** [comment, marked as answer]
Is Fabric API being loaded in the production environment? Your build script might be exposing Fabric API, including its resource loader, to only the development environment causing resources in the `assets` folder to only loaded there.

**Reply by hugeBlack** [reply]
I have fabric-loader and fabric-api in my build.gradle , so I don’t know what to do. Please see it [here](https://github.com/hugeBlack/McfDebugger_Mod/blob/master/build.gradle)

**Reply by haykam821** [reply]
It looks like you are not loading Fabric API in the production environment because you are only using `modImplementation` to declare it as a dependency. Does specifying `include` as well or loading Fabric API as a separate JAR work?

**Reply by hugeBlack** [reply]
I installed fabricapi and that worked for me. Thanks
